script.js

```diff
@@ -0,0 +1,142 @@
+                document.getElementById("dropDown").innerHTML += `<option value="farenheit">Farenheit</option>`;
+                document.getElementById("dropDown").innerHTML += `<option value="celsius">Celsius</option>`;
+                document.getElementById("dropDown").innerHTML += `<option value="kelvin">Kelvin</option>`;
+                document.getElementById("dropDown").innerHTML += `<option value="rankine">Rankine</option>`;
+        document.getElementById("inputRow").innerHTML += `<div id="dataEntryCol" class="col">`;
+            document.getElementById("dataEntryCol").innerHTML += `<label class="ml-4">Value</label><br>`;
+            document.getElementById("dataEntryCol").innerHTML += `<input type="text" id="tempInput">`;
+        document.getElementById("inputRow").innerHTML += `<div id="buttonCol" class="col">`;
+            document.getElementById("buttonCol").innerHTML += `<br><button id="tempButton" onclick ="farenheitDisplay(), fToC(), fToK(), fToR()" class="px-4 mt-2">Calculate</button>`;
+    document.getElementById("outerBorder").innerHTML += `<div id="outputRow" class="row d-flex justify-content-center mt-4">`;
+        document.getElementById("outputRow").innerHTML += `<div id="farenheitCol" class="col ml-5">`;
+            document.getElementById("farenheitCol").innerHTML += `<label><strong>Farenheit</strong></label><br>`;
+            document.getElementById("farenheitCol").innerHTML += `<p id="farenheitValue"></p>`;
+        document.getElementById("outputRow").innerHTML += `<div id="celsiusCol" class="col">`;
+            document.getElementById("celsiusCol").innerHTML += `<label><strong>Celsius</strong></label><br>`;
+            document.getElementById("celsiusCol").innerHTML += `<p id="celsiusValue"></p>`;
+        document.getElementById("outputRow").innerHTML += `<div id="kelvinCol" class="col">`;
+            document.getElementById("kelvinCol").innerHTML += `<label><strong>Kelvin</strong></label><br>`;
+            document.getElementById("kelvinCol").innerHTML += `<p id="kelvinValue"></p>`;
+        document.getElementById("outputRow").innerHTML += `<div id="rankineCol" class="col mr-5">`;
+            document.getElementById("rankineCol").innerHTML += `<label><strong>Rankine</strong></label><br>`;
+            document.getElementById("rankineCol").innerHTML += `<p id="rankineValue"></p>`;
+
+// Select and Button Logic
+let dropDown = document.getElementById("dropDown");
+let buttonCol = document.getElementById("buttonCol");
+function typeChange() {
+    if(dropDown.value == "farenheit") {
+        buttonCol.innerHTML = "";
+        buttonCol.innerHTML += `<br><button id="tempButton" onclick ="farenheitDisplay(), fToC(), fToK(), fToR()" class="px-4 mt-2">Calculate</button>`;
+    } else if(dropDown.value == "celsius") {
+        buttonCol.innerHTML = "";
+        buttonCol.innerHTML += `<br><button id="tempButton" onclick ="celsiusDisplay(), cToF(), cToK(), cToR()" class="px-4 mt-2">Calculate</button>`;
```

Comment:
It would be better to have the button always call one function, say `calculate()` and in the calculate method you determine what values to display and how to calculate them